src/accordion/Accordion.test.js

```diff
@@ -1,21 +1,75 @@
-import test from 'ava';
-import jsdom from 'jsdom';
+import { assert } from 'chai';
 import nunjucks from 'nunjucks';
+import Velocity from 'velocity-animate';
 
 import { Accordion } from './Accordion';
 
-test.beforeEach(t => {
-  global.document = jsdom.jsdom(nunjucks.render(`./Accordion.test.html`));
-});
+let accordion;
 
-// @todo: Write real tests
-test('Can initialize Accordion', t => {
-  Accordion.init('#accordion');
-  t.truthy(Accordion.accordion);
-  t.is(Accordion.transitionDuration, 300);
-});
+const isHidden = (el) => {
+  let style = window.getComputedStyle(el);
+
+  return (style.display === 'none');
+};
+
+describe('Accordion', () => {
+  before(() => {
+    fixture.setBase('src');
+
+  });
+
+  beforeEach(() => {
+    accordion = fixture.load('accordion/Accordion.test.html');
+
+    document.body.insertAdjacentHTML('afterbegin', accordion);
+  });
+
+  afterEach(() => {
+    accordion = document.querySelector('#accordion');
+    accordion.parentNode.removeChild(accordion);
+  });
+
+  it('should initialize', () => {
+    Accordion.init('#accordion');
+
+    assert.isDefined(Accordion.accordion);
+    assert.equal(Accordion.transitionDuration, 300);
+  });
+
+  it('should initialize with custom transitionDuration', () => {
+    Accordion.init('#accordion', 100);
+
+    assert.equal(Accordion.transitionDuration, 100);
+  })
+
+  it('should initialize with visible sectionContent', () => {
+    Accordion.init('#accordion');
+
+    const sectionContents = document.querySelectorAll('.js-accordion-section-content');
+
+    for (var i = 0; i < sectionContents.length; i++) {
```

Comment:
Just curious, would a `forEach` function work better here for the assertions of each sectionContent?